docs/trusted-resources.md

```diff
@@ -157,7 +154,11 @@ spec:
 To learn more about regex syntax please refer to [syntax](https://pkg.go.dev/regexp/syntax).
 To learn more about `ConfigSource` please refer to resolvers doc for more context. e.g. [gitresolver](./git-resolver.md)
 
- `key` is used to store the public key, note that secretRef and data cannot be configured at the same time.
+ `key` is used to store the public key, `key` can be configured with `secretRef`, `data`, `kms` note that only 1 of these 3 fields can be configured.
+
+  * `secretRef`: refers to secret in cluster to store the public
+  * `data`: contains the inline data of the pubic key
```

Comment:
Add that this is "PEM-encoded byte slice" format.

Reply:
sure! Thanks!